lib/gemstash/cli.rb

```diff
@@ -66,5 +67,20 @@ def status
     def stop
       Gemstash::CLI::Stop.new(self).run
     end
+
+    desc "preload", "Preloads all the gems in your gemstash server from the default upstream"
+    method_option :latest, :type => :boolean, :default => false, :desc =>
+      "Only fetch the latest specs"
+    method_option :server_url, :type => :string, :default => "http://localhost:9292", :desc =>
+      "Gemstash server url"
+    method_option :threads, :type => :numeric, :default => 20, :desc =>
+      "Number of threads to run the fetching job with"
+    method_option :limit, :type => :numeric, :default => nil, :desc =>
+      "Limit for the number of gems to preload"
+    method_option :skip, :type => :numeric, :default => 0, :desc =>
+      "Number of gems to skip when preloading"
+    def preload
```

Comment:
Hmmm, ok, I misunderstood this task a bit then. I fear this might be a bit confusing for a user... every other command on gemstash is oriented as a server side action, so this feels a bit out of place as a client action here (to me). Should we consider a separate client program, like what Stickler does? Or maybe re-orient the actions with a prefix... like `gemstash server ...` or perhaps only for client actions like `gemstash client preload ...`? Maybe that's excessive if this ends up as the only client side Gemstash action.

Thoughts?